data/repolist.txt

```diff
@@ -127,12 +125,10 @@ dotnet/roslyn branch=dev15.8-preview4-vs-deps server=dotnet-ci
 dotnet/roslyn branch=microupdate server=dotnet-ci
 dotnet/roslyn branch=features/codecov server=dotnet-ci
```

Comment:
@jaredpar if *branch count* is the source of the problem, this one can go too. It's not part of the auto-merges so it's just sitting until I get around to working on it again.